templates/base.html

```diff
@@ -56,22 +56,87 @@ <h1 class="brand m-0 h4"><a href="/">2021 Georgia Early Vote Ballot Tracker</a><
+            <h3 class='text-center'>Do your part: Help your friends in Georgia vote</h3>
+            <hr class='mb-4'>
+          </div>
+
+          <div>
+            <h4>1️⃣ Who do you know in Georgia?</h4>
+            <p>Check which of your Facebook friends lives in Georgia.</p>
+            <a href="https://www.facebook.com/search/top?q=my%20friends%20who%20live%20in%20georgia" target="_blank" rel="noopener" class="btn text-white mb-3" style="background-color: #e12814;">Find friends in Georgia on Facebook</a>
+            <p><em>Did you know it's possible to find your friends on Facebook with a text search? Click the button to run the search, we won't post anything to your wall or spam your friends.</em></p>
+            <!-- TODO: Add something about needing to be logged into FB, otherwise you get an error -->
+            <!-- TODO: Add something about what the search is, if they are hesitant to click the button -->
+            <!-- TODO: Deep link to FB on mobile? -->
+            <hr class='my-4'>
+          </div>
+
+          <div class='mt-4'>
+            <h4>2️⃣ Let them know there's an election</h4>
+            <p>In most elections, over 30% of voters don't know when the election is. Make sure your friends know that the election is on <strong>Tuesday, January 5, 2021</strong>. Anyone can vote early starting on <strong>Monday, December 14, 2020</strong>.</p>
+            <!-- TODO: Add image of copy for text and email, make "copyable" -->
+
+            <!-- TODO: See if this works, I have an Android rn -->
+            <a href="sms://body=I%27m%20sure%20you%20already%20know%20that%20the%20Georgia%20runoff%20election%2C%20the%20one%20that%20determines%20who%20controls%20the%20Senate%2C%20is%20on%20January%205.%20I%20just%20wanted%20to%20pass%20along%20this%20link%20to%20find%20your%20early%20voting%20location%20--%20you%20can%20start%20voting%20today%21%20https%3A%2F%2Fwww.voteamerica.com%2Fwhere-to-vote%2F" target="_blank" rel="noopener" class="btn text-white mb-3" style="background-color: #e12814;">Text a friend</a>
+            
+            <a href="mailto:?to=&subject=Vote%20early%20in%20the%20Georgia%20runoff%20election&body=I%27m%20sure%20you%20already%20know%20that%20the%20Georgia%20runoff%20election%2C%20the%20one%20that%20determines%20who%20controls%20the%20Senate%2C%20is%20on%20January%205.%20I%20just%20wanted%20to%20pass%20along%20this%20link%20to%20find%20your%20early%20voting%20location%20--%20you%20can%20start%20voting%20today%21%20https%3A%2F%2Fwww.voteamerica.com%2Fwhere-to-vote%2F" target="_blank" rel="noopener" class="btn text-white mb-3" style="background-color: #e12814;">Email a friend</a>
+            
+            <hr class='my-4'>
+          </div>
+
+          <div class='mt-4'>
+            <h4>3️⃣ Follow up!</h4>
+            <p>Research shows that people vote when you ask them to. Make sure to follow up and ask your friends if they've voted!</p>
+
+            {% block content %}{% endblock %}
```

Comment:
OOKOKOKOK i just saw this and wrapped my head around what is happening here. i think the key change i'd suggest is: instead of making these changes in base.html, it feels like this code mostly belongs inside of index.html. base.html is meant to be a common template shared across all the pages on the site. for example, you wouldn't want these steps to show up on the search results page or on the faq page. so i would suggest moving most of the code into index.html and revert base.html to basically be an empty page with the `{% block content %}{% endblock %}` in it.

Reply:
sure